btree/btree.c

```diff
@@ -980,6 +981,7 @@ struct nd {
 	int                     n_txref;
```

Comment:
A few lines above please add a blank line between the declaration of n_skip_rec_count_check and the comment following it. Also add a blank line between declaration of n_nref and the comment following it.

Reply:
Updated in latest commit.